detection-rules/attachment_adobe_image_lure_qr_code.yml

```
@@ -51,41 +51,44 @@ source: |
                   // recipient email address is present in the URL, a common tactic used in credential phishing attacks 
                   and any(recipients.to,
                           (
-                            strings.icontains(..scan.qr.data, .email.email)
-                            and (
-                              .email.domain.valid
-                              or strings.icontains(.display_name, "undisclosed")
+                            (
+                              strings.icontains(..scan.qr.data, .email.email)
+                              or any(beta.scan_base64(..scan.qr.data),
+                                     strings.icontains(., ..email.email)
+                              )
                             )
                           )
-  
-                          // the recipients sld is in the senders display name
-                          or any(recipients.to,
-                                 strings.icontains(sender.display_name,
-                                                   .email.domain.sld
-                                 )
+                          and (
+                            .email.domain.valid
+                            or strings.icontains(.display_name, "undisclosed")
                           )
+                  )
+              )
   
-                          // the recipient local is in the body  
-                          or any(recipients.to,
-                                 strings.icontains(body.current_thread.text,
-                                                   .email.local_part
-                                 )
-                          )
+              // the recipients sld is in the senders display name
+              or any(recipients.to,
+                     strings.icontains(sender.display_name, .email.domain.sld)
+              )
+  
+              // the recipient local is in the body  
+              or any(recipients.to,
+                     strings.icontains(body.current_thread.text,
+                                       .email.local_part
+                     )
+              )
```

loop recipients.to one less time
```suggestion
            or any(recipients.to,
                   // the recipients sld is in the senders display name
                   strings.icontains(sender.display_name, .email.domain.sld)
                   // the recipient local is in the body  
                   or strings.icontains(body.current_thread.text,
                                        .email.local_part
                   )
            )
```